front/src/applications/stdcmV2/components/StdcmResults.tsx

```diff
@@ -1,51 +1,52 @@
-import React, { useState } from 'react';
+import React, { useMemo, useState } from 'react';
 
+import { Button } from '@osrd-project/ui-core';
 import { CheckCircle } from '@osrd-project/ui-icons';
+import { PDFDownloadLink } from '@react-pdf/renderer';
 import { useTranslation } from 'react-i18next';
 
 import type { ManageTrainSchedulePathProperties } from 'applications/operationalStudies/types';
+import SimulationReportSheetV2 from 'applications/stdcm/components/SimulationReportSheetV2';
 import type { StdcmV2SuccessResponse } from 'applications/stdcm/types';
-import { generateCodeNumber } from 'applications/stdcm/utils';
-import type { RollingStockWithLiveries } from 'common/api/osrdEditoastApi';
+import { generateCodeNumber, getOperationalPointsWithTimes } from 'applications/stdcm/utils';
 import { Map } from 'modules/trainschedule/components/ManageTrainSchedule';
-import { formatDateToString } from 'utils/date';
+import { dateTimeFormatting } from 'utils/date';
 
-import StdcmTableResults from './StdcmTableResults';
+import StcdmResultsTable from './StdcmResultsTable';
 
 type StcdmResultsV2Props = {
   stdcmData: StdcmV2SuccessResponse;
   pathProperties?: ManageTrainSchedulePathProperties;
-  rollingStockData: RollingStockWithLiveries;
-  speedLimitByTag?: string;
-  simulationCreationDate?: Date;
 };
 
-const StcdmResults = ({
-  stdcmData,
-  pathProperties,
-  rollingStockData,
-  speedLimitByTag,
-  simulationCreationDate,
-}: StcdmResultsV2Props) => {
+const StcdmResults = ({ stdcmData, pathProperties }: StcdmResultsV2Props) => {
   const { t } = useTranslation('stdcm');
-  const date =
-    simulationCreationDate &&
-    t('formattedCreationDate', formatDateToString(simulationCreationDate));
+  const date = dateTimeFormatting(stdcmData.creationDate, false, 'alternate');
```

Comment:
When I read the code and I see false like this, it's a bit weird. Put it in the const (at the beginning of the file for instance) and use the const instead of false like this:
```suggestion
  const date = dateTimeFormatting(stdcmData.creationDate, withoutTime, 'alternate');
```

Reply:
fix